tests/unit/backend/data/test_Indexer.py

```diff
@@ -929,64 +927,64 @@ def test_readWriteRecord_normalization(self):
-        assert type(res) is Normalization
-        assert res == params
-
-    def test_readWriteParameters_reduction(self):
-        params = CalculationParameters(**DAOFactory.calibrationParameters().model_dump())
-        indexer = self.initIndexer(IndexerType.REDUCTION)
-        indexer.writeParameters(params)
-        res = indexer.readParameters(params.version)
-        assert type(res) is CalculationParameters
-        assert res == params
+    # def test_readParameters_nope(self):
+    #     indexer = self.initIndexer()
+    #     assert not indexer.parametersPath(1).exists()
+    #     with pytest.raises(FileNotFoundError):
+    #         indexer.readParameters(1)
+
+    # def test_readWriteParameters(self):
+    #     version = randint(1, 10)
+    #     params = self.calculationParameters(version)
+
+    #     indexer = self.initIndexer()
+    #     indexer.writeParameters(params)
+    #     res = indexer.readParameters(version)
+    #     assert res.version == version
+    #     assert res == params
+
+    # def test_readWriteParameters_warn_overwrite(self):
+    #     version = randint(1, 100)
+
+    #     indexer = self.initIndexer()
+
+    #     # write some parameters at a version
+    #     params1 = self.calculationParameters(version)
+    #     indexer.writeParameters(params1)
+    #     assert indexer.parametersPath(version).exists()
+
+    #     # now try to overwrite parameters at same version
+    #     # make sure a warning is logged
+    #     with self.assertLogs(logger=IndexerModule.logger, level=logging.WARNING) as cm:
+    #         params2 = self.calculationParameters(version)
+    #         indexer.writeParameters(params2)
+    #     assert f"Overwriting  parameters at {indexer.parametersPath(version)}" in cm.output[0]
+
+    # # make sure the indexer can read/write specific state parameter types #
+
+    # def test_readWriteParameters_calibration(self):
+    #     params = DAOFactory.calibrationParameters()
+    #     indexer = self.initIndexer(IndexerType.CALIBRATION)
+    #     indexer.writeParameters(params)
+    #     res = indexer.readParameters(params.version)
+    #     assert type(res) is Calibration
+    #     assert res == params
+
+    # def test_readWriteParameters_normalization(self):
+    #     params = DAOFactory.normalizationParameters()
+    #     indexer = self.initIndexer(IndexerType.NORMALIZATION)
+    #     indexer.writeParameters(params)
+    #     res = indexer.readParameters(params.version)
+    #     assert type(res) is Normalization
+    #     assert res == params
+
+    # def test_readWriteParameters_reduction(self):
+    #     params = CalculationParameters(**DAOFactory.calibrationParameters().model_dump())
+    #     indexer = self.initIndexer(IndexerType.REDUCTION)
+    #     indexer.writeParameters(params)
+    #     res = indexer.readParameters(params.version)
+    #     assert type(res) is CalculationParameters
+    #     assert res == params
```

Comment:
There was an oversight that would let parameters write w/o there being a valid index entry, it would just create its desired folders.  These tests are commented out because read/writeParameters just calls the same function as everything else now, so these test are redundant/will be rewritten in the context of `Save/LoadVersionedObject` if novel.